Classes/RedirectStorage.php

```diff
@@ -182,7 +182,7 @@ public function addRedirect(
                 $sourceUriPath,
                 $targetUriPath,
                 $statusCode,
-                [],
+                null,
```

Comment:
I think there is still a bug, when `$hosts` is `null` wouldn't it try to run `array_map` on it?

And in your commit message you reference `addRedirectByHost` and not `addRedirect`.

Reply:
The $hosts param of `addRedirect()` is always an array, if this it's an empty array, the above fix is needed because `addRedirectByHost()` expects a single $host string or null.

Reply:
Correct, was looking at the wrong lines...